### PR TITLE
Updated navigator installation method

### PR DESCRIPTION
**Pull request by ianf77**
Leaving ansible-autmoation-platform-2.1-for-rhel-8-x86_64-rpms enabled
causes installation to fail. A new installayion method was suggested.

Install Automation content navigator on RHEL Document update

https://issues.redhat.com/browse/AAP-4972

### CODE REVIEW

File: downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc [hunk 1]
```diff
@@ -17,27 +17,13 @@ You can install {Navigator} on Red Hat Enterprise Linux (RHEL) from an RPM.
 
 .Procedure
 
-. Attach the {PlatformName} SKU.
-+
-----
-$ subscription-manager attach --pool=<sku-pool-id>
-----
+. Install {Navigator} using the following:
```
Comment (oraNod) [inline downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc:20]:
```suggestion
. Install {Navigator} with the following command:
```

File: downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc [hunk 2]
```diff
@@ -17,27 +17,13 @@ You can install {Navigator} on Red Hat Enterprise Linux (RHEL) from an RPM.
...
 
-. Enable the repository for RHEL 8.
-+
+[options="nowrap" subs="+quotes"]
 ----
-$ sudo subscription-manager repos --enable ansible-automation-platform-2.1-for-rhel-8-x86_64-rpms
+sudo dnf install --enablerepo=ansible-automation-platform-2.2-for-rhel-8-x86_64-rpms ansible-navigator
 ----
```
Comment (oraNod) [inline downstream/modules/navigator/proc-installing-navigator-rhel-rpm.adoc:25]: I think it's a good thing to remove the dollar sign at the command prompt but is this consistent with the rest of the guide?

**Comment by ianf77** [comment]
I've asked Lucas Benedito to check, as he proposed the new method.

**Review by ariordan-redhat — APPROVED** [review]
LGTM

**Pull request merged**